### DISCUSSION TITLE
Just a 'thank you'

### DISCUSSION THREAD

**Discussion by stripedgoat** [yattee/yattee#60] [General]
I've tried Yattee about 2 months ago on my iPhone, and ended up uninstalling it after 5 mins of playing around with it (due to unpleasant UX).

Recently I thought I'd give it another chance, and this time I am pleasantly surprised with the improvements. 👏🏼 The overall UX is much better than it was the last time I tried it. It was mainly small things that I was really missing...For example, when you quit the full screen, the video continues playing, etc...Now the natural flow of using the app is quite great.

Thank you for developing this app 🙏🏼
Looking forward for the further improvements 😊

**Comment by arekf** [comment]
Thanks for your kind words.

I am a sole person working on this not-so-small project and I had to cut corners sometimes to be able to release anything. Most common UI issues that were reported are connected to the player (like the one you listed in your message - behavior of pausing on closing full screen is what Apple's player forces and I can only workaround by restarting playing).

I am continuously working on further improvements to the interface. Soon I will release a beta with new player component for 4K playback, new controls, gestures and overall better UI. It is now in alpha testing for iOS - if anyone's interested please join [project Matrix channel](https://matrix.to/#/#yattee:matrix.org) to download and discuss it.

And remember - this app will get better only if issues and suggestions are reported, I review each of them.

Cheers!

**Comment by stripedgoat** [comment]
Yeah, I understand it must not be easy to keep it all up... The mentioned features will be very welcome! (Especially gestures)

**Comment by redstrate** [comment]
I just wanted to add my thanks as well (without creating a new thread) This is the exact app I was looking forever for in the Apple ecosystem, but it's already made and it works **extremely well** out of the box too! I already sent some money your way, and I can't wait to try this app some more. I might send a PR or two as well ;-)

**Comment by FillingTheVoidOnYT** [comment]
thank you
